kernel/boot.s

```diff
@@ -1,47 +1,67 @@
-// SPDX-FileCopyrightText: CGL-KFS
-// SPDX-License-Identifier: BSD-3-Clause
-
-/* boot.s
- *
- * Boot sector of our kernel
- *
- * created: 2022/10/11 - lfalkau <lfalkau@student.42.fr>
- * updated: 2022/11/22 - mrxx0 <chcoutur@student.42.fr>
- */
-
```

Comment:
I think that this file should be way more commented as it's very important. I have no idea of what and how the bootstrap stuff works without more context, and as it's assembly, and for a tricky step of the project, you definitely need to avoid any confusion by adding comments.